lib/writer/enum_model_writer.dart

```diff
@@ -7,6 +7,8 @@ class EnumModelWriter {
 
   String write() {
     final sb = StringBuffer()
+      ..writeln("// GENERATED CODE - DO NOT MODIFY BY HAND")
```

Comment:
move this generated text to a const somewhere so we don't need to update it in both files if we want to change this